data/json/recipes/recipe_deconstruction.json

```
@@ -255,119 +255,119 @@
     "activity_level": "LIGHT_EXERCISE",
     "time": "1 m",
     "qualities": [ { "id": "HAMMER", "level": 1 } ],
-    "components": [ [ [ "garnet", 1 ] ] ]
+    "components": [ [ [ "silver_small", 2 ] ], [ [ "garnet", 1 ] ] ]
```

```suggestion
    "components": [ [ [ "garnet", 1 ] ] ]
```

I manually removed silver from those rings in the past because they weigh 10mg, while 2 silver weighs 10g i.e. 1000x more. This is a regression.

And... this seems to go for everything here? You're just kind of undoing most of #64568 here it looks like. This isn't good unless you want to change the item weights.